app/helpers/Emails/Notifications/AssignmentPoints/PointsChangedEmailsSender.php

```diff
@@ -127,32 +120,31 @@ public function shadowPointsUpdated(ShadowAssignmentPoints $points): bool {
     }
 
     $locale = $awardee->getSettings()->getDefaultLanguage();
-    $subject = $this->shadowPointsUpdatedPrefix .
-      EmailLocalizationHelper::getLocalization($locale, $points->getShadowAssignment()->getLocalizedTexts())->getName();
+    list($subject, $text) = $this->createShadowPointsUpdated($points, $locale);
```

Comment:
If the create function yielded `EmailRenderResult`, this is not the best way how to get the properties out (I do not even think this will work without explicit cast to array).